mscore/pianoroll/pianolevelschooser.ui

```diff
@@ -2,13 +2,8 @@
 <ui version="4.0">
  <class>PianoLevelsChooser</class>
  <widget class="QWidget" name="PianoLevelsChooser">
-  <property name="geometry">
-   <rect>
-    <x>0</x>
-    <y>0</y>
-    <width>369</width>
-    <height>288</height>
-   </rect>
+  <property name="windowTitle">
+   <string>Form</string>
```

Comment:
Please remove this string, it is not needed

Reply:
This is automatically inserted by the IDE whenever the form is edited.  I'll change the name to something the translators might find more understandable.